static/css/styles.css

```diff
@@ -4813,7 +4837,7 @@ a:hover, a:active, a:focus {
 
 ul li {
   line-height: 20px;
-  list-style-image: none;
+  list-style-image: url(http://osuosl.org/sites/wwwstaging.osuosl.org/themes/doug_fir/images/bullet.png);
```

Comment:
This should NOT be using a direct URL like this and instead should be a relative URL that points to a file hosted in this theme.